content/docs/command-reference/exp/diff.md

```diff
@@ -0,0 +1,39 @@
+# exp diff
+
+Show [parameters](/doc/command-reference/params) and
+[metrics](/doc/command-reference/metrics) changes between `dvc experiments`, or
+between the last <abbr>project</abbr> version and the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc experiments diff [-h] [-q | -v] [--all] [--show-json]
+                            [--show-md] [--old] [--no-path]
+                            [--precision <n>] [a_rev] [b_rev]
+
+positional arguments:
+  a_rev            Old experiment to compare (defaults to HEAD)
+  b_rev            New experiment to compare (defaults to the current
+                   workspace)
```

Comment:
BTW should these args be called something else? I guess technically they're Git revisions but that's a deep implementation detail...

Reply:
Also does this expect the entire short hashes (7-digit)? Or something like `exp-bfe64`? Or both? What about full SHA-1 hashes?

Reply:
It accepts git revisions (as either short or full SHA hashes), git branch/tag names, or experiment names (whether it's `exp-1234` or a user specified name via `dvc exp run -n/--name ...`). All of the other diff commands in DVC also accept this same list now (including experiment names) - so `metrics diff`, `params diff`, etc

Reply:
it's probably also worth nothing that accepting experiment names isn't specific to the diff commands either, basically anything in DVC that takes a `--rev` (that previously worked with git branch/tag names) should also now accept experiment names as well, since they are resolved into a git SHA internally exactly the same way that git branch/tag names are resolved.

Reply:
Sounds good. I'd still call them `a_exp` and `b_ext` but it's prob. also alright to leave them like this for consistency with existing `diff` commands.

> anything in DVC that takes a (local) --rev (and already worked with git branch/tag names) should also now accept experiment names as well

Interesting! So the names are Git labels in the `exps`? Good to know but indeed no need to doc that for now prob.

Reply:
exp names are just git refs, same as git branch names or git tags. The only difference is that they are not automatically resolved by CLI git, since CLI git only automatically resolves shortened refs for `refs/heads/<name>` (branches) and `refs/tags/<name>`.